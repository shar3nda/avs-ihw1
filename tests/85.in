10
9 2 2 8 3 6 8 10 9 4