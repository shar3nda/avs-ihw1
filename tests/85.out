0 1 4 9 -1 1 5 8